core/src/trezor/lvglui/scrs/app_passkeys.py

```
@@ -0,0 +1,334 @@
+
+    return find_all()
+
+
+def delete_credential(index: int) -> None:
+    from storage.resident_credentials import delete
+
+    return delete(index)
+
+
+class PasskeysGuide(lv.obj):
+    def __init__(self, title, process_bar_value, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)